week-2-wide-perspective/w02e03-pagination-problem/src/middlewares/pagination.js

```diff
@@ -0,0 +1,17 @@
+export const pagination =
+  (customSkip = "skip", customLimit = "limit") =>
+  (req, res, next) => {
+    const { [customSkip]: skip, [customLimit]: limit } = req.query;
+    let computedSkip = 0;
+    if (skip) {
+      computedSkip = Number(skip);
+    }
+    let computedLimit;
+    if (limit) {
+      computedLimit = Number(limit);
+    }
+    req.computedSkip = computedSkip;
+    req.computedLimit = computedLimit;
```

Comment:
ja bym to podłączył do jakiegoś `.pagination = {skip, limit}`

Wtedy by było wygodniej w `controller`:

```javascript
req.pagination
```

Reply:
ale to jest detal